src/libraries/System.Data.Common/tests/System/Data/SqlTypes/SqlStringSortingTest.cs

```diff
@@ -86,7 +87,16 @@ private static void SqlStringDefaultCompareOptionTest(int localeID)
             SqlString str1;
             SqlString str2;
 
-            for (int i = 0; i < s_specialMatchingString.GetLength(0); ++i)
+            int count = s_specialMatchingString.GetLength(0);
+
+            // Some of Windows versions have a regression, so ignore last entry in the s_specialMatchingString if this is the case.
+            if (PlatformDetection.IsWindows10Version1903OrGreater &&
+                CultureInfo.InvariantCulture.CompareInfo.Compare("\u3060", "\uFF80\uFF9E", CompareOptions.IgnoreKanaType | CompareOptions.IgnoreWidth | CompareOptions.IgnoreCase) != 0)
+            {
+                count--;
```

Comment:
you might also want to add a comment above the last test case that it's important that it's last because of this bug

Reply:
good idea!